cg/services/order_validation_service/errors/sample_errors.py

```diff
@@ -52,3 +52,8 @@ class OrganismDoesNotExistError(SampleError):
 class SampleNameNotAvailableError(SampleError):
     field: str = "name"
     message: str = "Sample name already used in previous order"
+
+
+class WellFormatError(SampleError):
+    field: str = "well_position"
+    message: str = "Well position is not in correct format"
```

Comment:
Would be nice to give the customers knowledge of how it should look like. Maybe "Well position must follow the format 'A-H:1-12'"?